src/views/index.html

```diff
@@ -369,9 +373,11 @@ <h1>GALLERY</h1>
 
     <footer>
       <div class="container">
+          <a href='#'>
         <img class="acc-logo" src="../static/images/logo-acc.png" />
+          </a>
```

Comment:
ok, I am now confused why you have it scroll up to the top.

Will the user expect this? Is this a common behaviour?

Reply:
oh yeah:o since the header doesn't stick or like pop back up when you scroll down, its for like fast navigation back to the top especially for like mobile devices where you have to physically scroll up to the top o-o

this will pretty much let them click the button and instead of refreshing, they'd be pulled back up to the top

Reply:
Like, I get what it DOES. But I was wondering if it was an industry standard? 

My internship in the summer kinda drilled into me "what does the user expect," so now Im traumatized 😅